SimG4Core/PhysicsLists/plugins/module.cc

```diff
@@ -20,14 +22,19 @@
 #include "QGSPCMS_FTFP_BERT_EML95.hh"
 #include "QGSPCMS_FTFP_BERT_EML_New.hh"
 #include "QGSPCMS_FTFP_BERT_EML95msc93.hh"
+#include "QGSPCMS_FTFP_BERT_EMM.hh"
```

Comment:
can I ask my question again? :)

> On Sep 5, 2016, at 11:33 AM, Malik Shahzad Muzaffar <notifications@github.com> wrote:
> 
> In SimG4Core/PhysicsLists/plugins/module.cc:
> 
> > @@ -20,14 +22,19 @@
> >  #include "QGSPCMS_FTFP_BERT_EML95.hh"
> >  #include "QGSPCMS_FTFP_BERT_EML_New.hh"
> >  #include "QGSPCMS_FTFP_BERT_EML95msc93.hh"
> > +#include "QGSPCMS_FTFP_BERT_EMM.hh"
> 
> @civanch , now it is missing QGSPCMS_FTFP_BERT_EMM.hh
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>